README.md

```diff
@@ -1,5 +1,12 @@
 [![AsyncAPI Generator](./assets/readme-banner.png)](https://www.asyncapi.com/tools/generator)
 
+This is a Monorepo managed using [Turborepo](https://turbo.build/) and contains the following package:
+
+1. [Generator](apps/generator): This is a tool that you can use to generate whatever you want basing on the AsyncAPI specification file as an input.
+
+2. [Generator-filters](apps/nunjucks-filters): This library contains generator filters that can be reused across multiple templates, helping to avoid redundant work. These filters are designed specifically for Nunjucks templates and are included by default with the generator, so there's no need to add then to  dependencies seprately.
```

Comment:
"no need to add them to  dependencies seprately." instead of "no need to add then to  dependencies seprately."

Reply:
done.